eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -0,0 +1,538 @@
+                case 'datetime':
+                    $modeTxt = 'Converting timestamps for ezdatetime fields.';
+                    break;
+            }
+
+            $output->writeln(
+                [
+                    $modeTxt,
+                    'Calculating number of Content Object Attributes to update...',
+                ]
+            );
+            $count = $this->countEzDateObjectAttributes();
+            $output->writeln(
+                [
+                    sprintf('Found total of Content Objects Atributes for update: %d', $count),
+                    '',
+                ]
+            );
+
+            if ($count == 0) {
+                $output->writeln('Nothing to process, exiting.');
+
+                return;
+            }
+
+            $helper = $this->getHelper('question');
+            $question = new ConfirmationQuestion(
+                '<question>Are you sure you want to proceed?</question> ',
+                false
+            );
+
+            if (!$helper->ask($input, $output, $question)) {
+                $output->writeln('');
+
+                return;
+            }
+
+            $progressBar = $this->getProgressBar($count, $output);
+            $progressBar->start();
+
+            for ($offset = 0; $offset < $count; $offset += $iterationCount) {
+                $processBuilder = new ProcessBuilder(
+                    [$this->getPhpPath(), $consoleScript, $this->getName(), $this->timezone]
+                );
+
+                if ($from) {
+                    $processBuilder->add('--from=' . $from);
+                }
+                if ($to) {
+                    $processBuilder->add('--to=' . $to);
+                }
+                $processBuilder->add('--mode=' . $this->mode);
+                $processBuilder->add('--offset=' . $offset);
+                $processBuilder->add('--iteration-count=' . $iterationCount);
+                $processBuilder->add('--env=' . $input->getOption('env'));
+                if ($this->dryRun) {
+                    $processBuilder->add('--dry-run');
+                }
+                $processBuilder->setEnv('INNER_CALL', 1);
+                $process = $processBuilder->getProcess();
+                $process->run();
+
+                if (!$process->isSuccessful()) {
+                    throw new RuntimeException($process->getErrorOutput());
+                }
+
+                $this->done += (int) $process->getOutput();
+
+                $progressBar->advance($this->done);
+            }
+
+            $progressBar->finish();
+            $output->writeln(
+                [
+                    '',
+                    sprintf('Done: %d', $this->done),
+                ]
+            );
+        }
+    }
+
+    /**
+     * @param int $offset
+     * @param int $limit
+     */
+    protected function processTimestamps($offset, $limit)
+    {
+        $ezDateObjectAttributes = $this->getEzDateObjectAttributes($offset, $limit);
```

Comment:
terminology: Fields is the word for Attributes in eZ Platform

Also you can drop the use of "Ez" in methods, looks strange, and can be omitted if we are specific as this is anyway inside eZ Platform.

example `$dateFields = $this->getDateFields($offset, $limit); `

And then adapt the rest of the method and other places for the simpler naming ;)